plugins/lighthouse-backend/README.md

```diff
@@ -19,13 +19,13 @@ import { PluginEnvironment } from '../types';
 import { CatalogClient } from '@backstage/catalog-client';
 
 export default async function createPlugin(env: PluginEnvironment) {
-  const { logger, scheduler, config } = env;
+  const { logger, scheduler, config, tokenManager } = env;
 
   const catalogClient = new CatalogClient({
     discoveryApi: env.discovery,
   });
 
-  await createScheduler({ logger, scheduler, config, catalogClient });
+  await createScheduler({ logger, scheduler, config, catalogClient tokenManager });
```

Comment:
```suggestion
  await createScheduler({ logger, scheduler, config, catalogClient, tokenManager });
```

Reply:
I like the idea to preserve the compatiblity with old contract. I'm going to implement this.